prediction_market_agent/agents/known_outcome_agent/known_outcome_agent.py

```diff
@@ -178,7 +182,10 @@ def has_question_event_happened_in_the_past(
         date_str=date_str,
         question=question,
     )
-    answer = str(llm.invoke(prompt).content)
+    config: RunnableConfig = {
+        "callbacks": [langfuse_context.get_current_langchain_handler()]
+    }
+    answer = str(llm.invoke(prompt, config=config).content)
```

Comment:
It's not clear to me what `callbacks` mean, since they are being passed to the LLM definition (see line 177) and then being overwritten when `llm.invoke` is called (line 188).

Reply:
Yep seems like they can be passed in both places and then they are merged:

![Screenshot by Dropbox Capture](https://github.com/user-attachments/assets/bbf3f062-9b7a-442b-9e85-6391dccd6862)

(this is from Langchain's code, where `callbacks` is from invoke's arguments, and `self.callbacks` is from the class)

I removed one of them, as we don't need both.